org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: webappreader.sh
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home  # to be able to save html pages to a hard drive
+  - --share=network # it is necessary to download html from the Internet
+  #- --device=dri # For the Qt webengine module to work perfectly with gpu
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess # necessary for QtWebEngineProcess to work
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - python3-requirements.json
+  
+  - name: wget2 #  proper workable version = 2.1
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap
+      - ./configure --prefix=/app
+      - make
+      - make install
+    sources:
+      - type: git
+        url: https://gitlab.com/gnuwget/wget2.git
+        commit: 657e3d22c488a2d1f173e208e253995e15f8b6db
+        
+  - name: webappreader
+    buildsystem: simple
+    build-commands: 
+      - mkdir -p /app/webappreader/
+      - cp -r database /app/webappreader/database
+      - cp -r pages /app/webappreader/pages
+      - cp -r src /app/webappreader/src
+      - cp -r ui /app/webappreader/ui
+      - install -Dm644 constants.py -t /app/webappreader
+      - install -Dm644 settings.py -t /app/webappreader
+      - install -Dm644 tools.py -t /app/webappreader
+      - install -Dm644 webappreader.py -t /app/webappreader
+    sources:
+      - type: git
+        url: https://gitverse.ru/sc/shekin/webappreader.git
+        commit: 88d98aa310103be2704a3908a0cec1aec65e8a01
+      - type: file
+        path: org.mintguide.WebAppReader.desktop
```

Comment:
this file doesn't exist.

Reply:
ohh))) yes.. not anouth rights)) i will move it to another hosting//.. thx

Reply:
this file still doesn't exist.